app/boards/arm/bdn9/Kconfig.defconfig

```diff
@@ -14,4 +14,8 @@ config ZMK_KEYBOARD_NAME
 config ZMK_USB
 	default y
 
+config ZMK_RGB_UNDERGLOW
+    select SPI
```

Comment:
Mixed tabs and spaces for these two indents as well.

Reply:
Good catches. Fixed both.